### DISCUSSION TITLE
How can k3s reduce the interaction with the database

### DISCUSSION THREAD

**Discussion by 983183947** [k3s-io/k3s#8063] [Q&A]
I am using an external database now. I found that k3s interacts frequently with the database. How can I set it up to reduce the interaction with the database?

**Comment by brandond** [comment]
The Kubernetes apiserver is constantly making requests against the datastore; there's not really anything to be done about that. While you could probably poke through the kubelet and controller documentation to find some intervals to increase to get marginal reductions in database traffic, I can't recommend doing that. The improvements will probably not be worth the degraded responsiveness.
